tests/plugins/pastefromword/parsestyles.html

```diff
@@ -39,7 +39,7 @@
 	mso-style-qformat:yes;
 	mso-style-parent:"";
 	margin:0cm;
-	margin-bottom:.0001pt;
+	margin-bottom:.001pt;
```

Comment:
Edge rounds `.0001` to 0 when parsing stylesheet so I changed it to `.001` which works fine.